game/graphics/renderer.cpp

```diff
@@ -187,7 +214,9 @@ void Renderer::initSettings() {
 
   auto prevVidResIndex = settings.vidResIndex;
   settings.vidResIndex = Gothic::inst().settingsGetF("INTERNAL","vidResIndex");
-  settings.fxaaEnabled = (Gothic::options().fxaaPreset > 0) && (settings.vidResIndex==0);
+  settings.cmaa2Enabled = (Gothic::options().cmaa2Preset>0) && (settings.vidResIndex==0);
+  settings.fxaaEnabled = (Gothic::options().fxaaPreset>0) && (settings.vidResIndex==0) && !settings.cmaa2Enabled;
```

Comment:
Is FXAA still relevant? Any case, when use can prefer FXAA to CMAA?

Reply:
Just in case someone prefer more blurry image. I doubt there many people with such preferences though.